deploy/kubernetes/overlays/prow-gke-release-staging-head/kustomization.yaml

```diff
@@ -7,14 +7,14 @@ images:
   newName: gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver
   newTag: "latest"
```

Comment:
hmmm this is a problem too isn't it?

Reply:
maybe we need to enable GCE_PD_DO_DRIVER_BUILD?

Reply:
yeah this tag is really old now - from sep 9.

Enabling the build flag isn't so easy - then we also need to add logic to go and parse this kustomization file and swap out "latest" for whatever the new build tag is... Though that logic already exists for the pre-submits so maybe its not so hard to port over to this file too. Want to do that or follow up later with it?

We should really figure out what to do about the test tag versions

Reply:
From a quick glance, I think our k8s-integration.go should be able to handle substituting the image properly when we set the flag. It takes [both overlay and registry](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/k8s-integration/driver.go#L15) as an argument. So no extra changes needed here.

Reply:
ack. are you adding that change to this PR?

Reply:
no, setting DO_DRIVER_BUILD is not in this repo. I'll send out a separate pr for that